src/main/steps/save-sign-out/get.ts

```diff
@@ -1,26 +0,0 @@
-import autobind from 'autobind-decorator';
-import { Response } from 'express';
-
-import { AppRequest } from '../../app/controller/AppRequest';
-import { GetController, Translations } from '../../app/controller/GetController';
-import { SAVE_SIGN_OUT_URL } from '../../steps/urls';
-
-import { saveAndSignOutContent } from './content';
-
-@autobind
-export class SaveSignOutGetController extends GetController {
-  constructor(protected readonly content: Translations = { ...saveAndSignOutContent }) {
-    super(__dirname + '/template', content);
-  }
-
-  public async get(req: AppRequest, res: Response): Promise<void> {
-    if (req.session.user?.email) {
-      res.cookie('nfdivEmail', req.session.user.email, { maxAge: 60000 });
-      req.session.destroy(() => res.redirect(SAVE_SIGN_OUT_URL));
```

Comment:
why not call super.get after the session was destroyed with the email as you do below? that way you dont need to add the cookie and also itll be separated.

I did something similar in my session timeout pr. [PR-276](https://github.com/hmcts/nfdiv-frontend/pull/276/files#diff-750d18e61d3f6d442bcd12bd8b6da8beee0c0defc962fcb834d1bfc7a144d3b0R17)

My only issue with this is super.get gets called twice but i still think its better to keep it separately.

Reply:
This line and the cookie logic have been removed. It now renders the "Signed out" on post via

<details>
<summary><a href="https://github.com/hmcts/nfdiv-frontend/pull/281/files#diff-c1f1642e840be0c399e414942589582f371e232d042e55010e227edea2247fb0R48">src/main/app/controller/PostController.ts</a></summary>

```ts
  private saveAndSignOut(req: AppRequest<T>, res: Response) {
    const email = req.session.user?.email;
    const language = req.session?.lang || 'en';

    req.session.destroy(err => {
      if (err) {
        throw err;
      }

      res.render(`${__dirname}/../../steps/save-sign-out/template.njk`, {
        ...commonContent[language],
        ...saveAndSignOutContent[language],
        email,
      });
    });
  }
```
</details>

Reply:
yh, i saw that but i just thought, it might be better to have it as a separate controller where you destroy with the commands below of updates the email property and then call super.get